sushi-bot/index.js

```diff
@@ -299,13 +311,32 @@ module.exports = (clients) => {
 				dailyAsaCounter.max(user, best_score);
 			}
 		}
+
+		{
+			if(text.includes(":exercise-done:")||text.includes(":kintore_houkoku:")){
+				await increment(user, 'exercise-cumulative');
+				Promise.resolve()
+				.then(() => slack.reactions.add({name: 'erai', channel, timestamp}))
+				.then(() => slack.reactions.add({name: 'sugoi', channel, timestamp}))
```

Comment:
[MAY] 単にメインの実行をブロッキングしたくない (awaitしたくない) 意図なら

```suggestion
				slack.reactions.add({name: 'erai', channel, timestamp}))
				slack.reactions.add({name: 'sugoi', channel, timestamp}))
```
のほうがシンプルでわかりやすいと思います。

Reply:
unhandled promise rejection~~~

```suggestion
				Promise.all([
				  slack.reactions.add({name: 'erai', channel, timestamp}),
				  slack.reactions.add({name: 'sugoi', channel, timestamp}),
				])
```

Reply:
あ、awaitしたくないのか (日本語が苦手)